src/RichText/UserMention.php

```diff
@@ -204,6 +204,10 @@ public static function getUserIdsFromUserMentions(string $content)
             return [];
         }
 
+        if (!$content_as_xml || is_null($content_as_xml)) {
```

Comment:
`simplexml_import_dom()` will return `null` on failure, not `false`.

```suggestion
        if ($content_as_xml === null) {
```